client/reducers/accountInfo.js

```diff
@@ -0,0 +1,66 @@
+import axios from 'axios';
+
+// ACTION TYPES
+const GET_USER_INFO = 'GET_USER_INFO';
+const UPDATE_USER_INFO = 'UPDATE_USER_INFO';
+const DELETE_USER = 'DELETE_USER';
+
+// ACTION CREATORS
+export const getUserInfo = userInfo => {
+  return {type: GET_USER_INFO, userInfo};
+}
+export const updateUserInfo = updatedUser => {
+  return {type: UPDATE_USER_INFO, updatedUser}
+}
+
+export const deleteUser = user => {
+  return {type: DELETE_USER, user}
+}
+
+// REDUCER
+export default (user = {}, action) => {
+  switch (action.type) {
+    case GET_USER_INFO:
+      return action.userInfo
+    case UPDATE_USER_INFO:
+      return Object.assign({}, user, action.updatedUser)
+    case DELETE_USER:
+      return {}
+    default:
+      return user
+  }
+}
+
+//THUNK
+export const fetchUserInfo = () => {
+  return dispatch => {
+    return axios.get('/api/account')
```

Comment:
think the route is wrong

Reply:
Not sure why you think the route is wrong, it looks right to me.  Can you show me?

Reply:
sorry seeing things.  its right